h2o-py/h2o/model/word_embedding.py

```diff
@@ -39,4 +40,12 @@ def transform(self, words, aggregate_method):
         :returns: the approximate reconstruction of the training data.
         """
         j = h2o.api("GET /3/Word2VecTransform", data={'model': self.model_id, 'words_frame': words.frame_id, 'aggregate_method': aggregate_method})
-        return h2o.get_frame(j["vectors_frame"]["name"])
\ No newline at end of file
+        return h2o.get_frame(j["vectors_frame"]["name"])
+
+    def to_frame(self):
+        """
+        Converts a given word2vec model into H2OFrame.
+
+        :returns: a frame representing learned word embeddings.
+        """
+        return h2o.H2OFrame._expr(expr=ExprNode("word2vec.to.frame", self))
```

Comment:
Capability API makes sense, it should be applied on a larger unit, eg. for the whole word2vec algo. Rapid "word2vec.to.frame" should always be available if word2vec itself is available. I think we don't need to check in individual methods like in this one.

We could probably check just in the constructor of H2OWord2vecEstimator, what do you think?